src/tonstakers.ts

```diff
@@ -112,6 +119,7 @@ class Tonstakers extends EventTarget {
     this.isTestnet = wallet.account.chain === BLOCKCHAIN.CHAIN_DEV;
 
     this.walletAddress = Address.parse(wallet.account.address);
+    this.hexWalletAddress = wallet.account.address
```

Comment:
we can use `this.walletAddress.toRawString()` instead of making a new attribute. This could backfire if we forget to reset this address somewhere (which is actually what happened in this PR, `deinitialize` doesn't reset the address for some reason)

Reply:
done